sdks/java/core/src/main/java/org/apache/beam/sdk/testing/WindowFnTestUtils.java

```diff
@@ -67,14 +68,30 @@
   public static <T, W extends BoundedWindow> Map<W, Set<String>> runWindowFn(
       WindowFn<T, W> windowFn,
       List<Long> timestamps) throws Exception {
```

Comment:
Rather than using `List<Long>` to represent timestamps, we should probably use `List<Instant>` so the types are more explicit?